packages/wallet-sdk/src/libInfo.ts

```diff
@@ -0,0 +1,2 @@
+export const LIB_VERSION = '4.1.0';
```

Comment:
can we rename this file to `sdk-info.ts`

Reply:
updated!